### PR TITLE
Update format of evaluated value shown in the debugger

### LINKED ISSUE #4761: [Bug] Update format of evaluated value in debugger
~~Table Data Evaluated Value~~
--------------------------
~~{~~
  
~~}~~

{
  [propertyPath]: [value]
}

### PR DESCRIPTION
**Pull request by akash-codemonk**
## Description
The error messages shown for widget property validation errors has been formatted.

Fixes #4761 

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Manually

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: feat/format-evaluate-value 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 51.65 **(-0.02)** | 32.85 **(-0.05)** | 29.58 **(-0.01)** | 52.13 **(-0.02)**
 :red_circle: | app/client/src/components/editorComponents/Debugger/Errors.tsx | 52.94 **(-3.31)** | 36.36 **(-30.31)** | 0 **(0)** | 56.25 **(-3.75)**
 :red_circle: | app/client/src/components/editorComponents/Debugger/LogItem.tsx | 33.33 **(-5.38)** | 15.38 **(-4.62)** | 0 **(0)** | 33.33 **(-5.65)**
 :red_circle: | app/client/src/sagas/ActionExecutionSagas.ts | 12.16 **(0)** | 1.26 **(-0.03)** | 7.14 **(0)** | 13.95 **(0)**</details>

### CODE REVIEW

File: app/client/src/components/editorComponents/Debugger/LogItem.tsx [hunk 1]
```diff
@@ -182,6 +197,49 @@ function LogItem(props: LogItemProps) {
     },
     collapsed: 1,
   };
+  let displayValue = null;
+
+  switch (getType(props.value)) {
```
Comment (rishabhsaxena) [inline app/client/src/components/editorComponents/Debugger/LogItem.tsx:202]: Can we add comments here as examples for cases where this is applicable?

**Review by rishabhsaxena — CHANGES_REQUESTED** [review]

**Review by rishabhsaxena — APPROVED** [review]
image

This looks a bit redundant since we already have the path in the message currently?

File: app/client/src/components/editorComponents/Debugger/LogItem.tsx [hunk 2]
```diff
@@ -197,6 +242,8 @@ function LogItem(props: LogItemProps) {
     dispatch(toggleShowGlobalSearchModal());
   }, []);
 
+  console.log(props.state, "props.state");
```
Comment (rishabhsaxena) [inline app/client/src/components/editorComponents/Debugger/LogItem.tsx:245]: Should rm log

**Comment by akash-codemonk** [comment]
Closing this as requirement was updated. Doing the necessary change in #5144

**Pull request closed without merging**